docs/topics/azure-stack.md

```diff
@@ -97,30 +95,30 @@ Unless otherwise specified down below, standard [cluster definition](../../docs/
 
 | Name                            | Required | Description|
 | ------------------------------- | -------- | ---------- |
-| vmsize                          | yes      | Describes a valid [Azure Stack VM size](https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-vm-sizes). |
+| vmsize                          | yes      | Describes a valid [Azure Stack VM size](https://docs.microsoft.com/azure-stack/user/azure-stack-vm-sizes). |
 | distro                          | yes      | Specifies the masters' Linux distribution. Currently supported values are: `"ubuntu"` and `"aks"`. The latter is a custom image based on ubuntu-16.04 that comes with pre-installed software necessary for Kubernetes deployments. `"aks"` should be selected if you target a disconnected Azure Stack instance. |
 | availabilityProfile             | yes      | Only `"AvailabilitySet"` is currently supported. |
 | acceleratedNetworkingEnabled    | yes      | Use `Azure Accelerated Networking` feature for Linux agents. This property should be always set to `"false"`. |
 
-## Azure Stack Instances Registered with Azure's China cloud
+## Azure Stack instances registered with Azure's China cloud
 
 If your Azure Stack instance is located in China, then the `dependenciesLocation` property of your cluster definition should be set to `"china"`. This switch ensures that the provisioning process fetches software dependencies from reachable hosts within China's mainland.
 
-## Disconnected Azure Stack Instances
+## Disconnected Azure Stack instances
 
 _Note: AKS Engine on disconnected Azure Stack instances is a private preview feature_
 
 By default, the AKS Engine provisioning process relies on an internet connection to download the software dependencies required to create or upgrade a cluster (Kubernetes images, etcd binaries, network plugins and so on).
 
-If your Azure Stack instance is air-gapped or if network connectivity in your geographical location is not reliable, then the default approach will not work, take a long time or timeout due to transient networking issues.
+If your Azure Stack instance is air-gapped or if network connectivity in your geographical location is not reliable, then the default approach will not work due to the length of time for the download or a timeout due to transient networking issues.
```

Comment:
1 If air-gapped, then default approach does not work at all
2 if not reliable network, then it may fail due timeout or transient issues

I think the rewrite does not capture # 1